src/components/description/styles.module.css

```diff
@@ -0,0 +1,11 @@
+.descriptionContainer {
+    margin:0 auto;
+    background-color: var(--page-bg);
+}
+
+.descriptionContent {
+    text-align: justify;
```

Comment:
don't keep it justified, it should be either center or left (which is default)

Reply:
done